neursafe_fl/python/client/client_reporter.py

```diff
@@ -149,6 +149,7 @@ async def start(self):
                             "not join the fl system.")
             return
 
+        self.__task_manager.set_client_id(self.id)
```

Comment:
Id belongs to client self, not reporter.
So shouldn't set client id in reporter.